### ISSUE TITLE
Use last committee size instead of default

### ISSUE THREAD

**Issue by girazoki** [witnet/witnet-rust#1546]

**Comment by tmpolaczyk** [comment]
You can use this test, but make sure that the expected values are what we actually want

```rust
    #[test]
    fn test_current_committee_size_requirement_sequence() {
        let default_size = 100;
        let decreasing_period = 5;
        let decreasing_step = 2;
        let mut last_consolidated_checkpoint = 0;
        let mut current_checkpoint = 0;
        let mut size = 0;

        let mut next_size = |has_superblock| {
            let s = current_committee_size_requirement(default_size, size, decreasing_period, decreasing_step, last_consolidated_checkpoint, current_checkpoint);
            if has_superblock {
                last_consolidated_checkpoint = current_checkpoint;
                size = s;
            };
            current_checkpoint += 1;
            s
        };

        // Check that the committee size is 100 during the first epochs if there are superblocks
        let mut initial = vec![];
        for _ in 0..10 {
            initial.push(next_size(true));
        }
        assert_eq!(initial, vec![100; 10]);

        // Check the decreasing pattern from 100 with period 5 and step 2
        let mut decreasing = vec![];
        for _ in 0..30 {
            decreasing.push(next_size(false));
        }
        assert_eq!(decreasing, vec![100, 100, 100, 100, 98, 98, 98, 98, 98, 96, 96, 96, 96, 96, 94, 94, 94, 94, 94, 92, 92, 92, 92, 92, 90, 90, 90, 90, 90, 88]);

        // Set the current committee size back to 1
        for i in 0.. {
            if next_size(false) == 1 {
                break
            }

            if i == 1000 {
                panic!("Never reached commitee size 1");
            }
        }

        // Check the increasing pattern from 1 to 100 with step 2
        let mut increasing = vec![];
        for _ in 0..55 {
            increasing.push(next_size(true));
        }
        assert_eq!(increasing, vec![1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31, 33, 35, 37, 39, 41, 43, 45, 47, 49, 51, 53, 55, 57, 59, 61, 63, 65, 67, 69, 71, 73, 75, 77, 79, 81, 83, 85, 87, 89, 91, 93, 95, 97, 99, 100, 100, 100, 100, 100]);

        // Set the current committee size back to 1
        for i in 0.. {
            if next_size(false) == 1 {
                break
            }

            if i == 1000 {
                panic!("Never reached commitee size 1");
            }
        }

        // Check the increasing pattern from 1 to 100 with step 2
        let mut increasing = vec![];
        for _ in 0..3 {
            increasing.push(next_size(true));
            for _ in 0..decreasing_period {
                increasing.push(next_size(false));
            }
        }
        assert_eq!(increasing, vec![1, 3, 3, 3, 3, 1, 1, 3, 3, 3, 3, 1, 1, 3, 3, 3, 3, 1]);
    }
```

**Comment by tmpolaczyk** [comment]
Fixed in #1554
